internal/pkg/scaffold/gopkgtoml_test.go

```diff
@@ -91,5 +91,5 @@ required = [
   [[prune.project]]
     name = "k8s.io/kube-state-metrics"
     unused-packages = true
-  
+
```

Comment:
Any reason for this?

Reply:
My text editor removes extraneous whitespaces. I can revert this but it shouldn't be there to begin with.